Dockerfile.d/test-integration-soci-snapshotter.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=soci snapshotter containerd plugin for integration test
+Documentation=https://github.com/awslabs/soci-snapshotter
+After=network.target
+Before=containerd.service
+
+[Service]
+Type=notify
+ExecStart=/usr/local/bin/soci-snapshotter-grpc
```

Comment:
You can add `ExecStartPre` for the mount command in the unit file instead of Dockerfile.

```
ExecStartPre=/bin/bash -c 'mkdir -p /var/lib/soci-snapshotter-grpc && mount -t tmpfs none /var/lib/soci-snapshotter-grpc'
```